staging/src/k8s.io/pod-security-admission/test/fixtures_dropCapabilities.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/pod-security-admission/api"
+)
+
+/*
+TODO: include field paths in reflect-based unit test
+containerFields: []string{
+	`securityContext.capabilities.drop`,
+},
+*/
+
+func init() {
+	fixtureData_1_22 := fixtureGenerator{
+		expectErrorSubstring: "drop all",
+		generatePass: func(p *corev1.Pod) []*corev1.Pod {
+			return []*corev1.Pod{p}
+		},
+		generateFail: func(p *corev1.Pod) []*corev1.Pod {
+			p = ensureCapabilities(p)
+			return []*corev1.Pod{
```

Comment:
generateFail should return:

- pod that doesn't drop any capabilities
- a variety of pods dropping different capabilities, but not _all_ of them
- pods that drop all capabilities but add different prohibited capabilities back (i.e. stuff besides `CAP_NET_BIND_SERVICE`)

Reply:
yeah, those three testcases sound right to me